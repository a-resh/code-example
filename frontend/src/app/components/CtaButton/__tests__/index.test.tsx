import * as React from 'react';
import { render } from '@testing-library/react';

import { CtaButton } from '..';

describe('<CtaButton  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <CtaButton background={'black'} color={'white'} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
