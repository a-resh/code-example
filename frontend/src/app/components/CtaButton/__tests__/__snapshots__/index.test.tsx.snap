// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CtaButton  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 100%;
  width: 100%;
  color: white;
  background-color: black;
  cursor: pointer;
}

.c0 b {
  font-size: 20px;
  font-weight: 700;
}

<div
  class="blocks__Center-sarv4y-2 c0"
  color="white"
>
  <b>
    _NOT_TRANSLATED_
  </b>
</div>
`;
